<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SupportShortcuts</class>
 <widget class="QWidget" name="SupportShortcuts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QGroupBox" name="groupBox_play">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>9</y>
     <width>431</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Play control shortcuts</string>
   </property>
   <widget class="QTextEdit" name="play_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>411</width>
      <height>170</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_other">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>213</y>
     <width>431</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Other control shortcuts</string>
   </property>
   <widget class="QTextEdit" name="other_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>411</width>
      <height>111</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
